### ISSUE TITLE
A11y_AzureDataStudioAzureArcExtension_AzureArcController_ConnecttoExistingAzureArcController_ScreenReader: Nvda is not announcing the required field "Kube config file path" as required.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15732]
### Actual:
Nvda is not announcing the required field "Kube config file path" as required.

Similar issue is observed with "Jaws"

### Expected:
Nvda should be announcing the required field "Kube config file path" as required.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AzureArcExtension_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#NVDA;#JAWS;#STP;;#WCAG1.3.1;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio -Azure Arc Extension
Windows Version: Win10
Nvda version: 2020.4, Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to "Create new Azure Arc Controller"
4. Tab to "Kube config file path" edit box start nvda and check if it announces it as required field

### User Impact:
Screen Reader users will not get the proper information about the edit box

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by Shivamd1** [comment]
Closing as issue not repro

**Comment by Shivamd1** [comment]
![image](https://user-images.githubusercontent.com/60880975/150949286-63d5a050-b8b1-45a0-b42e-795d53da635d.png)

**Comment by prabhudevu** [comment]
**GithubTags:**#Closed;
